uaclient/config.py

```diff
@@ -792,17 +793,25 @@ def _get_entitlement_information(
 
     def simulate_status(
         self, token: str, show_beta: bool = False
-    ) -> Dict[str, Any]:
+    ) -> Tuple[Dict[str, Any], int]:
         """Return a status dictionary based on a token."""
```

Comment:
Let's just update the docstring for the new return.
Also, instead of returning the tuple directly, we could use a `NamedTuple` here to make the code easier to read on cli.
But that is just an style change, feel free to disagree

Reply:
I changed the docstring to explain why the int and what it means - then I think it is explanatory enough